Utils/main.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+
+from helper_functions import find_median, open_file, plot_histogram
+
+
+# define the main function for the program to run from here
+def main(
+    file_path: str,
+    feature_name: str,
+    image_name="image_name",
+    median=True,
+    histogram=True,
+    bins=50,
+):
+
+    # open the file and return the data in a pandas dataframe
+    df = open_file(file_path)
+
+    # find the median of the feature in the dataframe and return the value and index of the row
+    if median:
+        median, median_index = find_median(df, feature_name)
+        image_name = df[29:30].values[-1:].tolist()[0].pop()
+        plot_histogram(
+            df=df,
+            feature_name=feature_name,
+            image_name_column="image_name",
+            row_name=image_name,
+            histogram_=histogram,
+            bins=bins,
+        )
+    else:
+        plot_histogram(
+            df,
+            feature_name,
+            image_name_column="image_name",
+            row_name=image_name,
+            histogram_=histogram,
+            bins=bins,
+        )
```

Comment:
Depending on the intent, consider moving `image_name` and the first call to `plot_histogram` outside of the if branch and removing the second call to `plot_histogram`.